sdk/scheduler/src/main/java/com/mesosphere/sdk/framework/OfferProcessor.java

```diff
@@ -227,6 +230,41 @@ private void processQueuedOffers() {
         }
     }
 
+    private boolean checkStatuses() {
```

Comment:
The method name is a bit confusing to me.  Maybe we could change it to something starting with the word `is`.  Maybe something like `isActive`.